docs/pages/guide/01-start.md

```diff
@@ -1,6 +1,122 @@
 ---
 layout: guide
 group: guide
-title: Getting Started
+title: Product Overview
 ---
-## Welcome to Navi
+
+## <span class="c5">Yavin overview</span>
+
+<img src="assets/images/Yavin_logo.png" width="200"/>
+
+Yavin is a framework for rapidly building custom data applications that
+offers both a UI and an API. In addition, Yavin can be deployed as a
+standalone Business Intelligence tool. It is <span class="c1">scalable
+and secure</span> and it’s <span
+class="c1">intuitive</span> visualization interface allows for <span
+class="c1">fast drill-downs</span> of very large data sets through
+simple interface and visualization blocks. With Yavin, you can easily
+and intuitively create <span class="c1">reports</span>, which focus on
+arbitrary drill downs for a single visualization, or <span
+class="c1">dashboards</span>, which are collections of visualizations,
+for <span class="c1">rapidly distilling information</span><span
+class="c0">. </span>
+
+<span class="c0"></span>
+
+Disclaimer : For the sample demos that come with Yavin, we are using the
+“<span
+class="c17"><a href="https://www.google.com/url?q=https://www.kaggle.com/shivamb/netflix-shows&amp;sa=D&amp;ust=1606182367630000&amp;usg=AOvVaw0vjjwg_hSHcq80D3AjOpp1" class="c3">Netflix Movie and TV Shows</a></span>”
+data set that is sourced from <span
+class="c17"><a href="https://www.google.com/url?q=https://www.kaggle.com/&amp;sa=D&amp;ust=1606182367630000&amp;usg=AOvVaw2MS0pMaHmAABG0JmQABGOV" class="c3">Kaggle</a></span><span
+class="c0"> (Timestamped 11/2020). The use of this data is only for
+demonstrating the ease of onboarding any dataset to Yavin. And has no
+correlation with Yavin as a product or an Open Source Software.</span>
+
+<span class="c0"></span>
+
+<span class="c0">Yavin is a combination of three major modules.  They
+are:</span>
+
+- <span class="c0 c15">Denali (The design layer) </span>
```

Comment:
Design is a loaded term.  I wonder if we should qualify what we mean by design here.  I would probably describe this as: "The user interface design language".